src/components/kytos/misc/Menubar.vue

```diff
@@ -13,7 +14,7 @@
                :key="component.name"
                :icon="component.icon"
                :tooltip="component.tooltip"
-               @click.native="setItem(index+1)"/>
+               @click.native="setItem(index+1)"/>           
```

Comment:
```suggestion
               @click.native="setItem(index+1)"/>
```

Removing extra space.